compiler/noirc_frontend/src/hir/comptime/interpreter.rs

```diff
@@ -44,6 +44,8 @@ mod builtin;
 mod foreign;
 mod unquote;
 
+pub(crate) type ValueAndLocation = (Value, Location);
+
```

Comment:
```suggestion
```
I don't think the name `ValueAndLocation` adds anything over `(Value, Location)`